Updated ManogoHud.conf with the recent values

Faalagorn opened the pull request:
Fixes #92

flightlessmango commented:
I don't understand why the colors are written twice and I don't think it works to write `#` in the color.
Otherwise it looks good

Faalagorn commented:
Thanks for pointing that out! I copied that section from the comment without thinking. I now fixed it and changed it to big letters to match that of crosshair and readme.
